### DISCUSSION TITLE
How to multistream using owncast

### DISCUSSION THREAD

**Discussion by Bluscream** [owncast/owncast#3434] [Q&A] [answered]
I would like owncast to transcode my incoming stream and distribute it to twitch, youtube and steam. Basically what restream.io does but selfhosted and "free"

**Comment by gabek** [comment]
That is not what Owncast is.

I've had multiple ask for this exact same thing in the past, however. No offense, but I'm curious, what makes you think that Owncast, a standalone, open source, free, decentralized, independent video server should be integrating with the Big Tech services Owncast exists to be an alternative of?

It's like telling Instagram you wish your timeline could up on Twitter, or that you wish the vegan restaurant could cook you a steak. That's not what they do.

Enough people have asked for this that I'm guessing something in how Owncast is documented must be making people think that this is what Owncast is. I just have no idea what it could be. I'd love to get your insight, so I can improve it and be more clear about what Owncast is.

**Reply by Bluscream** [reply]
Well, streaming is all about exposure, so streaming to as many platforms as possible (including selfhosted) increases visibility, and therefor chances for success. Ideally "i" would prefer everyone to use my own site to watch my stream, but in reality thats not how the world of content creation works, sadly. However, offering a owncast instance for the viewers that do want to go the extra mile, is nice.

In my case however, i was more interested in the multistreaming part than the "having your own stream frontend" part, but the options for non-sucking stream splitters (that provide transcoding functionality) seem to be non-existant, so most of my searches using google and phind brought me here (user is searching something niche about streaming? yeah owncast is close enough, lmao)

**Comment by gabek** [comment]
I get where you're coming from if you think about streaming being only the "Twitch style" of content. But many (maybe most?) Owncast instances are actually private, where exposure is the opposite of what they want. They have a need for live streaming with their company, organization, application, event, etc. and Owncast gives them the ability to do so.

I just want to clarify in case somebody else comes upon this thread who isn't trying to do "Twitch style" streaming, not every use case is the same.

**Comment by saschafoerster** [comment, marked as answer]
@Bluscream What you are searching is maybe this tool: https://github.com/datarhei/restreamer
